tuf/client/updater.py

```diff
@@ -29,24 +29,28 @@
 
   An overview of the update process:
 
-  1. The software update system instructs TUF to check for updates.
+  0. The software update system instructs TUF to check for updates.
```

Comment:
All comments in README.md above apply to `tuf/client/updater.py` as well. Maybe we should de-duplicate? E.g. just point to the README.md here? What do others think?